Assets/Scripts/Characters/Titan/PlayerTitan.cs

```diff
@@ -20,6 +20,12 @@ public class PlayerTitan : MindlessTitan
         protected override void Awake()
         {
             base.Awake();
+
+            // TODO: Make the other camera modes work.
+            // This also unfortunately sets the preferred
+            // camera mode. But that will change when we fix it.
+            GameCursor.CameraMode = CameraMode.TPS;
```

Comment:
Try commenting out this line and see what happens. I'm not even sure what the intended behaviour is for player titans. I don't think it makes sense to actually have the normal camera modes apply to titans, but setting the camera mode like this sets the preferred camera mode, which messes with the player's settings...